app/components/Admin/ChangeRequest.js

```diff
@@ -16,23 +14,23 @@ class ChangeRequest extends React.Component {
 
   componentDidMount() {
     this.retrieveModifiedObject();
-    let fields = this.props.changeRequest.field_changes;
-    let tempChangeRequestFields = {};
-    fields.forEach(field => {
+    const fields = this.props.changeRequest.field_changes;
+    const tempChangeRequestFields = {};
+    fields.forEach((field) => {
       tempChangeRequestFields[field.field_name] = field.field_value;
     });
 
     this.setState({ changeRequestFields: tempChangeRequestFields });
-
   }
 
   retrieveModifiedObject() {
-    let changeRequest = this.props.changeRequest;
-    let resource = changeRequest.resource;
-    //"ChangeRequest" is 13 characters, so this will give us the first part of the string
-    let objectType = changeRequest.type;
+    const changeRequest = this.props.changeRequest;
+    const resource = changeRequest.resource;
+    // "ChangeRequest" is 13 characters, so this will give us the first part of the string
+    const objectType = changeRequest.type;
+    let resourceNotes;
```

Comment:
The eslint rule `no-case-declarations` demands that variables are instantiated in the parent block. Agreed it feels messy when it is just assigned once.

Shall I make the rule a warning?